src/mame/shared/xbox_usb.cpp

```diff
@@ -75,12 +77,10 @@ uint32_t ohci_usb_controller::read(offs_t offset)
 {
 	uint32_t ret;
 
-#ifdef LOG_OHCI
 	if (offset >= 0x54 / 4)
-		logerror("usb controller 0 register HcRhPortStatus[%d] read\n", (offset - 0x54 / 4) + 1);
+		m_maincpu->machine().logerror("usb controller 0 register HcRhPortStatus[%d] read\n", (offset - 0x54 / 4) + 1);
 	else
-		logerror("usb controller 0 register %s read\n", usbregnames[offset]);
-#endif
+		m_maincpu->machine().logerror("usb controller 0 register %s read\n", usbregnames[offset]);
```

Comment:
The logging here is no longer conditional.

Reply:
Restored conditional logging using `if (LOG_OHCI)` to avoid dead code in the future.